docs/usage/creatingNewVariablesWithPython.rst

```diff
@@ -0,0 +1,35 @@
+.. _creatingNewVariablesWithPython:
+
+Creating new variables with python
+----------------------------------
+
+Vapor allows users to create new variables based on the data that exist in their files.  Users need to select input variables that will be read in their script, and they will need to define an output variable.  If the script successfully run by the Python Engine, the output variable will be usable in the same way as the native variables are in the dataset.
+
+Vapor allows users to create new variables to visualize with customizable Python scripts.  These variables are derived from data that is currently loaded into Vapor.  Additionally, Vapor comes with a set of convenience functions that allow users to easily compute new variables.
```

Comment:
These first two sentences just repeat what was said in the previous paragraph, don't they?

Reply:
True.  Thanks and fixed.